lib/data-ingestion-stack.ts

```diff
@@ -45,12 +46,13 @@ export class RealtimeDataIngestionStack extends Stack {
       timeout: Duration.seconds(60),
     });
 
+    const eventDetailType = 'Incoming Data';
     const inputStream = new EventbridgeToKinesisFirehoseToS3(this, 'InputStream', {
       eventBusProps: { eventBusName: `${this.prefix}-ingestion-bus` },
       eventRuleProps: { 
         ruleName: `${this.prefix}-ingestion-rule`,
         eventPattern: {
-          detailType: ["Incoming Data"],
+          detailType: [eventDetailType],
         },
       },
       kinesisFirehoseProps: { 
```

Comment:
Would it make sense to put `enabled: false` on line 62, so that the processing doesn't fail as long the `filter_duplicates` lambda function isn't implemented yet?

Reply:
Good point

Reply:
@leosch1 Done. Thanks again for pointing that out.